app/package.json

```diff
@@ -8,7 +8,7 @@
     "node": ">=18.0.0 <19.0.0"
   },
   "scripts": {
-    "start": "npx blitz@2.0.10 start --hostname 127.0.0.1 --port 3000",
+    "start": "npx blitz@2.0.10 start --hostname 127.0.0.1 --port 4000",
```

Comment:
FYI diese Änderungen müssen auch in den OAuth Apps (einmal Staging, einmal Production) nachgezogen werden.
Das habe ich jetzt schon gemacht, bspw. https://master.apis.dev.openstreetmap.org/oauth2/applications/521 (Zugang in Bitwarden)